raytracing.py

```diff
@@ -86,8 +86,7 @@ def trace_ray(rayO, rayD):
     toL = normalize(L - M)
     toO = normalize(O - M)
     # Shadow: find if the point is shadowed or not.
-    l = [intersect(M + N * .0001, toL, obj_sh)
-            for k, obj_sh in enumerate(scene) if k != obj_idx]
+    l = [intersect(M + N * .0001, toL, obj_sh) for k, obj_sh in enumerate(scene) if k != obj_idx]
```

Comment:
what a magic constant? Please define it as constant or as a parameter

Reply:
Fixed, it was moved as a constant to the top of the file. Also added it's description.